src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
replace to the process.env:
`export const BASE_URL = 'https://mate.academy/students-api/';`